lib/main_common.pm

```diff
@@ -635,7 +635,12 @@ sub load_jeos_tests {
     }
 
     load_boot_tests();
-    loadtest "jeos/firstrun";
+    if (check_var('FIRST_BOOT_CONFIG', 'combustion')) {
+        loadtest 'microos/verify_setup';
+        loadtest 'microos/image_checks';
```

Comment:
Might work in general, (without combustion) if the condition for the flag file is adjusted

Reply:
We would need to set the variable in all tests or remove the default value over here.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/microos/image_checks.pm#L38

So as of now, I have added this condition

Reply:
I am honestly considering adding a check to find the first empty drive in order to fix this test suite for other archs within this PR

Reply:
> We would need to set the variable in all tests or remove the default value over here.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/microos/image_checks.pm#L38

Yep:

```
if (get_var("FIRST_BOOT_CONFIG", is_jeos ? "wizard" : "combustion+ignition") =~ /combustion/) {
```

If this needs to be done in other places some `uses_combustion` wrapper would work as well, but so far that doesn't seem necessary.

Reply:
Arguably it's actually the MicroOS/SLEM default which is wrong, as `wizard` is the default everywhere. It's just that MicroOS tests were introduced without `FIRST_BOOT_CONFIG` set so to avoid having to add that everywhere it defaults to `combustion+ignition`.

Reply:
> I am honestly considering adding a check to find the first empty drive in order to fix this test suite for other archs within this PR

I'm really wondering whether this is not a product bug. The order should be predictable and consistent IMO.

Reply:
https://bugzilla.opensuse.org/show_bug.cgi?id=1157133

Reply:
Ok. If the kernel doesn't have guarantees regarding order and there's no way to somehow query for a specific tag, then there's no way around looking for the empty one explicitly.